harness/determined/common/experimental/__init__.py

```diff
@@ -13,6 +13,8 @@
     ValidationMetrics,
 )
 from determined.common.experimental.project import Project
+from determined.common.experimental._util import OrderBy
```

Comment:
bleh. This file shouldn't exist (who is supposed to use it?). I guess I prefer not adding to it? But I don't know, given that it does maybe it's good to keep updated.